kymatio/scattering1d/tests/test_scattering1d.py

```diff
@@ -277,3 +277,46 @@ def test_scattering_shape_input():
         # should invoke the else branch
     assert "1-tuple" in ve.value.args[0]
     assert "integer" in ve.value.args[0]
+
+def test_batch_shape_agnostic():
+    J, Q = 3, 8
+    length = 1024
+    shape = (length,)
+
+    length_ds = length/2**J
```

Comment:
I know PEP8 allows this, but lots of linters don't get it and ask for spaces between operators